pkg/object/httpserver/mux.go

```diff
@@ -250,7 +250,7 @@ func (mi *muxInstance) serveHTTP(stdw http.ResponseWriter, stdr *http.Request) {
 
 	startAt := fasttime.Now()
 
-	span := mi.tracer.NewSpanWithStart(stdr.Context(), mi.superSpec.Name(), startAt)
+	span := mi.tracer.NewSpanForHTTP(stdr.Context(), mi.superSpec.Name(), stdr)
```

Comment:
Although in `NewSpanForHTTP` call the `fasttime.Now()`, but timestamp in `span` is now different with `startAt` (which is used in metric and accesslog below). Is this ok?

Reply:
yeah, and please help delete function `NewSpanWithStart`